src/server/log.js

```diff
@@ -14,7 +14,6 @@ function error(error) {
     }
     console.log(chalk.red.bold('SIM ERROR: ' + error));
     config.server && config.server.close();
-    process.exit(1);
```

Comment:
Exiting makes sense for standalone taco-simulate on the command-line, but programmatic consumers might prefer to not have taco-simulate kill them unexpectedly and instead deal with any errors themselves.

@dlebu should this at least re-throw the error so that programmatic consumers be aware that a fatal error occurred? You left the call to `server.close()`, which means the whole thing will be in a bad state.